Resources/Prototypes/Entities/Structures/Windows/shuttle.yml

```diff
@@ -51,6 +51,11 @@
     trackAllDamage: true
     damageOverlay:
       sprite: Structures/Windows/cracks.rsi
+  - type: DeltaPressure
+    minPressure: 15000
+    minPressureDelta: 10000
+    scalingType: Linear
+    scalingPower: 0.0005
```

Comment:
Are shuttle windows _supposed_ to have the same thresholds as normal reinforced windows?

Reply:
Oh that's a really cool feature! It's a literal fail safe.